docs/reference/commandline/swarm_join_token.md

```diff
@@ -11,7 +11,7 @@ parent = "smn_cli"
 # swarm join-token
 
 ```markdown
-Usage:	docker swarm join-token [--rotate] (worker|manager)
+Usage:	docker swarm join-token [OPTIONS] (worker|manager)
```

Comment:
Or we should update the usage of the command line to be more consistent with the other command, @thaJeztah wdyt ? :angel: